admin/query/src/main/java/org/codice/ditto/replication/admin/query/ReplicationUtils.java

```diff
@@ -89,10 +92,15 @@ public ReplicationSiteField updateSite(String id, String name, AddressField addr
     return getSiteFieldForSite(site);
   }
 
-  private String addressFieldToUrlString(AddressField address) {
+  private String addressFieldToUrlString(AddressField address, @Nullable String rootContext) {
+    String context = rootContext;
+    if (context == null) {
+      context = "services";
+    }
```

Comment:
❗️ ❓ it looks like we are ignoring the context in some cases. Won't that be confusing?

Reply:
I guess the context will only be used if host/port are passed in to construct the url. If a url is passed in then we will just use it as is. I don't have a good explanation as to why we support both ways but our UI only uses the host/port/context option.